scripts/install_mhnserver_unattended.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+if test -f "../server/config.py"; then
+    ./install_mhnserver.sh
```

Comment:
That makes sense, there's other calls in the script that require the script to be run in the scripts directory. Either way, good idea.